bin/castor

```diff
@@ -3,24 +3,10 @@
 
 use Castor\Console\ApplicationFactory;
 
-$argv = $_SERVER['argv'];
+$runPhar = getenv('CASTOR_RUN_PHAR');
```

Comment:
Does it work with `$_SERVER['CASTOR_RUN_PHAR']`? If yes, could you update the code? It's safer than getenv()

Reply:
not sure have to check, will change if it's work with that